File: C++/1021 - Banknotes and Coins.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>
using namespace std;
 
int main() {
    float dinheiro;
    int notas[] = {10000, 5000, 2000, 1000, 500, 200};
    int moedas[] = {100, 50, 25, 10, 5, 1};
    
    cin >> dinheiro;
    dinheiro = dinheiro * 100;
    

    cout << "NOTAS:\n";
    for (int i : notas) {
        printf("%d nota(s) de R$ %.2f\n", (int) dinheiro/i, (float) (i/100));
        dinheiro = fmod(dinheiro, i);
    }
    
    
    cout << "MOEDAS:\n";
    for (int i : moedas) {
        printf("%d moeda(s) de R$ %.2f\n", (int) dinheiro/i , ((float) i/100));
        dinheiro = fmod(dinheiro, i);
    }

    return 0;
}

File: Python/1021 - Banknotes and Coins.py
dinheiro = float(input())
notas = [100, 50, 20, 10, 5, 2]
moedas = [1.0, 0.50, 0.25, 0.10, 0.05, 0.01]

print("NOTAS:")

for i in notas:
    print(f"{int(dinheiro / i)} nota(s) de R$ {i:.2f}")
    dinheiro %= i

print("MOEDAS:")

for i in moedas:
    print(f"{int(dinheiro / i)} moeda(s) de R$ {i:.2f}")
    dinheiro = round(dinheiro % i, 2)
